src/Controller/AboutController.php

```diff
@@ -152,17 +152,27 @@ public function earlyCareerAction(Request $request) : Response
+            new Paragraph('The community behind eLife – including the research funders who support the journal, the editors and referees who run the peer-review process, and our Early-Career Advisory Group – are keenly aware of the pressures faced by early-stage investigators. That’s one reason we’re working to create a more positive publishing experience that will, among other things, help early-career researchers receive the recognition they deserve.'),
             new Paragraph('eLife supports and showcases early-career scientists and their work in a number of ways:'),
             new DefinitionList([
-                '<a href="'.$this->get('router')->generate('magazine').'">Magazine features</a>' => 'Early-career researchers and issues of concern to them are regularly featured in interviews, podcasts and articles in the Magazine section of eLife',
-                '<a href="'.$this->get('router')->generate('events').'">#ECRWednesday Webinars</a>' => 'A platform for the early-career community to share opportunities and explore issues around building a successful research career, on the last Wednesday of every month',
-                '<a href="'.$this->get('router')->generate('events').'">Events</a>' => 'We offer speakers and funding for scientific meetings organised for and by early-career researchers',
-                '<a href="'.$this->get('router')->generate('community').'">Travel grants</a>' => 'eLife offers funding to help early-career scientists get exposure and recognition for their work among leading scientists in their fields',
+                'Early Career Advisory Group' => 'eLife has invited a group of talented graduate students, postdocs and junior group leaders from across the world to our <a href="'.$this->get('router')->generate('about-people', ['type' => 'early-career']).'">Early-Career Advisory Group</a>. The ECAG acts as a voice for early-career researchers (ECRs) within eLife, representing their needs and aspirations and helping to develop new initiatives and shape current practices to change academic publishing for the better.',
+            ]),
+            new Paragraph('The role of the ECAG includes:'),
+            Listing::unordered([
+                'Offering ideas and advice on new and ongoing efforts with the potential to help early-career scientists',
+                'Providing direct support for ongoing programs, such as monthly webinars',
+                'Leading efforts to reach out to early-stage colleagues, to gather their feedback and/or connect them to the network',
+                'Participating in online or in-person events about issues of concern to early-stage researchers',
+                'Attending quarterly phone calls and an annual in-person meeting',
+            ], 'bullet'),
+            new Paragraph('For more information, take a look at this <a href="'.$this->get('router')->generate('inside-elife-article', ['id' => '140901c3']).'">video message</a> from the ECAG.'),
```

Comment:
Could 'from the ECAG' be included in the link too? Bit more descriptive than just 'video message'.

Reply:
Yes - Rowena's happy with that (may I prevail upon you to make that change, please?).